challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -44,6 +48,18 @@ export default {
   data: () => ({
     engagementCenterEnabled: eXo.env.portal.engagementCenterEnabled,
     tab: null,
+    isAdministrator: false,
   }),
+  created() {
+    this.init();
+  },
+  methods: {
+    init() {
+      document.dispatchEvent(new CustomEvent('displayTopBarLoading'));
+      this.$realizationsServices.getAllRealizationsStatus()
```

Comment:
After a second thought, I think that we may need to add this flag from JSP, as we do in https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/WEB-INF/jsp/peopleList.jsp#L15 to avoid blink effect of displaying a Third Tab just after few milliseconds of other tabs display.

Reply:
okay, I am on it !